docs/assets/jailbreak/chatgpt4.pngZone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]

```

Comment:
What are these file types doing?

Reply:
I am not entirely sure, they were automatically added when I built the website locally